modules/cli/src/main/scala/scala/cli/commands/export0/Export.scala

```diff
@@ -129,17 +136,22 @@ object Export extends ScalaCommand[ExportOptions] {
       sys.exit(1)
     }
 
-    val shouldExportToMill = options.mill.getOrElse(false)
-    val shouldExportToSbt  = options.sbt.getOrElse(false)
-    if (shouldExportToMill && shouldExportToSbt) {
+    val shouldExportToMill  = options.mill.getOrElse(false)
+    val shouldExportToSbt   = options.sbt.getOrElse(false)
+    val shouldExportToMaven = options.maven.getOrElse(false)
+
+    val exportOptions = List(shouldExportToMill, shouldExportToSbt, shouldExportToMaven)
+
+    if (exportOptions.count(identity) > 1) {
       logger.error(
         s"Error: Cannot export to both mill and sbt. Please pick one build tool to export."
       )
       sys.exit(1)
     }
 
     if (!shouldExportToJson) {
-      val buildToolName = if (shouldExportToMill) "mill" else "sbt"
+      val buildToolName =
+        if (shouldExportToMill) "mill" else if (shouldExportToSbt) "sbt" else "maven"
```

Comment:
```suggestion
        if (shouldExportToMill) "mill" else if (shouldExportToMaven) "maven" else "sbt"
```
SBT should stay as the default.